File: 프로그래머스/0/181874. A 강조하기/A 강조하기.cs
using System;

public class Solution 
{
    public string solution(string myString) 
    {
        char[] str = myString.ToCharArray();
        
        for(int i=0; i<str.Length; i++) {
            if(str[i] == 'a') {
                str[i] = 'A';
            }
            else if (str[i] != 'a' && str[i] != 'A') {
                str[i] = Char.ToLower(str[i]);
            }
        }
        string strA = new string(str);
        
        return strA;
    }
}

File: 프로그래머스/0/181874. A 강조하기/A 강조하기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string myString) 
{
    for(char &ch : myString)
    {
        if(ch == 'a')
        {
            char upperch = toupper(ch);
            ch = upperch;
        }
        else if(ch != 'A' && isupper(ch))
        {
            char lowerch = tolower(ch);
            ch = lowerch;
        }
    }
    return myString;
}